GitFyle.Core.Api/Brokers/Loggings/ILoggingBroker.cs

```diff
@@ -11,5 +11,7 @@ public interface ILoggingBroker
         void LogInformation(string message);
         void LogTrace(string message);
         void LogDebug(string message);
+        void LogWarning(string message);
```

Comment:
Remove LogWarning.  The scope of this PR is only to add LogCritical.   When we werge changes in from main, then LogWarning will be added as part of the incoming changes from main.